CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 ### Fixed
 * Label ListUsers API calls [#2000](https://github.com/openfga/openfga/pull/2000)
 
+### Changed
+* Refactor Check benchmark test to remove server dependency [#2017](https://github.com/openfga/openfga/pull/2017)
+
```

Comment:
Normally, we don't mention refactors in the changelog. You can add the `Skip-Changelog` label to this PR.